docs/development.md

```diff
@@ -64,9 +60,19 @@
 
 ## Development notes
 
+- Set version numbers:
+    ```sh
+    $ export PRESIDIO_LABEL=[version number]
+    $ export PRESIDIO_DEPS_LABEL=[version number]
+    ```
+- Set registry name:
+    ```sh
+    $ export DOCKER_REGISTRY=[registry login server]
+    ```
 - Build the bins with `make build`
-- Build the the Docker image with `make docker-build`
-- Push the Docker images with `make docker-push`
+- Build the basecontainers with `make docker-build-deps DOCKER_REGISTRY=${DOCKER_REGISTRY} PRESIDIO_DEPS_LABEL=${PRESIDIO_DEPS_LABEL}`
```

Comment:
Is registry mandatory? are there any defaults to mention?

Reply:
the default is ours, i think it should be removed from code. 
thoughts?

Reply:
fixed, added "optional" information